transform/snowflake-dbt/models/finance/arr_transactions.sql

```diff
@@ -46,6 +51,26 @@ with mrr as (
     where ls.issued_date is not null
         and ls.edition in ('Cloud Professional')
         and es.reason is null
+)*/
+
+with mrr as (
+   select
+    customer as stripeid,
+    customers.name as customer_name,
+    subscriptions.id as subscription_id,
+    cws_dns as customer_dns,
+    cws_installation as license_id,
```

Comment:
I am not sure about this. @sri-byte might now more.

Reply:
Thanks for confirming @sri-byte